assets/scripts/game.js

```diff
@@ -0,0 +1,131 @@
+const Player = () => {
+    let marker = "X"
+    const getMarker = () => marker
+    const switchPlayer = () => {
+        if(marker === "X") {
+            marker = "O"
+        } else {
+            marker = "X"
+        }
+    }
+    return { marker, switchPlayer, getMarker }
```

Comment:
We don't need to export `marker` since we already have the getter method `getMarker` here. If we examine the code, there wasn't any place where we called `marker`.